app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,230 @@
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/<runtime-group-id>/core-entities/services \
+  -- data '{
+    "name": "classic_pizzas",
+    "retries": 5,
+    "protocol": "http",
+    "host": "example.com",
+    "port": 80,
+    "path": "/classic_pizzas",
+    "connect_timeout": 6000,
+    "write_timeout": 6000,
+    "read_timeout": 6000,
+    "tags": [
+      "user-level"
+    ],
+    "client_certificate": {
+      "id": "CERTIFICATE_ID"
+    },
+    "tls_verify": true,
+    "tls_verify_depth": null,
+    "ca_certificates": [
+      "CA_CERTIFICATE_ID"
+    ],
+    "enabled": true
+  }'
+```
+1. Create the `Classic Pizzas` API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products \
+  --data '{
+    "id": "GATEWAY_SERVICE_ID",
+    "name": "Classic Pizzas",
+    "description": "API product for classic pizzas",
+    "portal_ids": "PORTAL_ID",
+    "labels": {
+        "env": "test"
+    }
+  }'
+```
+1. Create the `v1` version for the `Classic Pizzas` API product and publish it:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/api-products/<api-product-id>/product-versions \
+  --data '{
+    "id": "GATEWAY_SERVICE_ID",
+    "name": "v1",
+    "gateway_service": {
+        "runtime_group_id": "RUNTIME_GROUP_ID"
+    },
+    "publish_status": "published",
+  }'
+```
+{% endnavtab %}
+{% endnavtabs %}
+
+### Create developer teams
+
+Next, let's create two developer teams: "Delivery Partners" with API Viewer and Consumer permissions and "New Partners" with API Consumer permissions.
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon dev-portal %} **Dev Portal**.
+1. Click **Teams** in the sidebar and then click **Add team**.
+1. Enter the following:
+    * **Team name:** Delivery Partners
+    * **Description:** Team of vetted delivery partners that can view and consume APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Classic Pizzas** product from the dropdown menu and select **API Consumer** and **API Viewer** from the **Add roles** dropdown menu.
+1. Click **Teams** in the sidebar and click **New Team** again and enter the following:
+    * **Team name:** New Partners
+    * **Description:** Team of unvetted partners that can only view APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Classic Pizzas** product from the dropdown menu and select **API Viewer** from the **Add roles** dropdown menu. 
+{% endnavtab %}
+{% navtab API %}
+{:.note}
+> **Note:** Make sure you replace the placeholders in the following API requests with information specific to your environment. 
```

Comment:
I think we could remove the note and instead have a sentence that says like 
"this variable was returned in this part of the guide"
I also think we should link to the API specs somewhere here in case they need more info.